ocs_ci/ocs/pgsql.py

```diff
@@ -338,6 +340,44 @@ def respin_pgsql_app_pod(self):
             resource=app_pod, state=constants.STATUS_RUNNING, timeout=300
         )
 
+    def export_pgoutput_to_googlesheet(self, pg_output, sheet_index):
+        """
+        Collect pgbench output to google spreadsheet
+
+        Args:
+            pg_output (list):  pgbench outputs in list
+            sheet_name (str): Name of the sheet
+            sheet_index (int): Index of sheet
+
+        """
+        # Collect data and export to Google doc spreadsheet
+        g_sheet = GoogleSpreadSheetAPI(
```

Comment:
How are we going to store the results of multiple runs over multiple builds/releases? Is it:
1) one sheet per release with many tabs - each tab for a particular run
 - looks compact
2) one sheet for all the releases with many tabs - each tab for a particular run
 - will have have too many tabs
3) one sheet for each run?
 - will create too many individual files - difficult to browse through and compare
4) any other option?

Reply:
In the spreadsheet "E2E Workloads - PGSQL", I am maintaining individual tabs for each pgsql test. For every run the results of each test gets appended to its own tab. 

Below is the snap of pg_out data of test_pgsql.py for two runs,
![multiple_runs](https://user-images.githubusercontent.com/26133916/83851654-4f5d3a00-a730-11ea-90a5-feba2e4c984e.png)

We can collect this way for every run in the interim builds and once we have the final benchmark results we can move the final run details to a new tab and lock(this has to be done manually).

Reply:
I have updated the patch to only export test_pgsql.